tests/tx/test_verification.py

```diff
@@ -0,0 +1,448 @@
+
+        verify_weight_wrapped = Mock(wraps=block.verify_weight)
+        verify_reward_wrapped = Mock(wraps=block.verify_reward)
+
+        with (
+            patch.object(MergeMinedBlock, 'verify_weight', verify_weight_wrapped),
+            patch.object(MergeMinedBlock, 'verify_reward', verify_reward_wrapped),
+        ):
+            self.manager.verification_service.verify_basic(block)
+
+        # Block methods
+        verify_weight_wrapped.assert_called_once()
+        verify_reward_wrapped.assert_called_once()
+
+    def test_merge_mined_block_verify_without_storage(self) -> None:
+        block = self._get_valid_merge_mined_block()
+
+        verify_pow_wrapped = Mock(wraps=block.verify_pow)
+        verify_no_inputs_wrapped = Mock(wraps=block.verify_no_inputs)
+        verify_outputs_wrapped = Mock(wraps=block.verify_outputs)
+        verify_data_wrapped = Mock(wraps=block.verify_data)
+        verify_sigops_output_wrapped = Mock(wraps=block.verify_sigops_output)
+
+        verify_aux_pow_wrapped = Mock(wraps=block.verify_aux_pow)
+
+        with (
+            patch.object(MergeMinedBlock, 'verify_pow', verify_pow_wrapped),
+            patch.object(MergeMinedBlock, 'verify_no_inputs', verify_no_inputs_wrapped),
+            patch.object(MergeMinedBlock, 'verify_outputs', verify_outputs_wrapped),
+            patch.object(MergeMinedBlock, 'verify_data', verify_data_wrapped),
+            patch.object(MergeMinedBlock, 'verify_sigops_output', verify_sigops_output_wrapped),
+            patch.object(MergeMinedBlock, 'verify_aux_pow', verify_aux_pow_wrapped),
+        ):
+            block.verify_without_storage()
+
+        # Block methods
+        verify_pow_wrapped.assert_called_once()
+        verify_no_inputs_wrapped.assert_called_once()
+        verify_outputs_wrapped.assert_called_once()
+        verify_data_wrapped.assert_called_once()
+        verify_sigops_output_wrapped.assert_called_once()
+
+        # MergeMinedBlock methods
+        verify_pow_wrapped.assert_called_once()
+
+    def test_merge_mined_block_verify(self) -> None:
+        block = self._get_valid_merge_mined_block()
+
+        verify_pow_wrapped = Mock(wraps=block.verify_pow)
+        verify_no_inputs_wrapped = Mock(wraps=block.verify_no_inputs)
+        verify_outputs_wrapped = Mock(wraps=block.verify_outputs)
+        verify_data_wrapped = Mock(wraps=block.verify_data)
+        verify_sigops_output_wrapped = Mock(wraps=block.verify_sigops_output)
+        verify_parents_wrapped = Mock(wraps=block.verify_parents)
+        verify_height_wrapped = Mock(wraps=block.verify_height)
+
+        verify_aux_pow_wrapped = Mock(wraps=block.verify_aux_pow)
+
+        with (
+            patch.object(MergeMinedBlock, 'verify_pow', verify_pow_wrapped),
+            patch.object(MergeMinedBlock, 'verify_no_inputs', verify_no_inputs_wrapped),
+            patch.object(MergeMinedBlock, 'verify_outputs', verify_outputs_wrapped),
```

Comment:
It's missing a wrapped call to the `verify_number_of_outputs()` which is usually called by `verify_outputs()`.

Reply:
Thanks!